examples/cloudflare-workers/http-api/wrangler.toml

```diff
@@ -1,7 +1,8 @@
 name = "momento-cloudflare-worker-http"
 main = "src/worker.ts"
 compatibility_date = "2023-07-10"
+keep_vars = true
```

Comment:
There's a web-sdk one as well but i can fast follow for that. thanks for doing this!